src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -101,6 +101,14 @@ a#editUncommonSettingsButton:hover{
     cursor: hand;
 }
 
+.btn.btn-enable-mouse-events {
+    pointer-events: auto;
+}
+
+.btn.btn-enable-mouse-events:active {
```

Comment:
Is the `.btn` necessary? seems rather unusual

Also maybe add comments to explain the behavior associated with these two CSS rules